service/src/main/java/bio/terra/batchanalysis/controllers/BatchApiController.java

```diff
@@ -0,0 +1,21 @@
+package bio.terra.batchanalysis.controllers;
+
+import bio.terra.terra_batch_analysis.generated.api.BatchApi;
+import java.util.Collections;
+import java.util.List;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+
+@Controller
+@Slf4j
+public class BatchApiController implements BatchApi {
+
+  @Override
+  public ResponseEntity<List<String>> listMethods() {
+    // TODO: Get an answer from the database?
+    return ResponseEntity.ok(
+        Collections.singletonList(
```

Comment:
With JDK 11+ I tend to prefer the new collection constructors over the old ones in `Collection`. In this case you can write
```suggestion
        List.of(
```

Reply:
Funnily enough, IntelliJ corrected my original use of `List.of` to this `singletonList` construct, on account of having only one item

Reply:
Maybe the IDE's language level is incorrect? I have the opposite behavior in my setup. The intention I see is `Java > Collections > Replace with `Arrays.asList()`. (The name is misleading as it suggests the newer `List.of()` form if you have the IDE set to JDK 9 or later.)